direnvrc

```diff
@@ -103,11 +103,24 @@ _nix_argsum_suffix() {
   fi
 }
 
+_any_file_in_dir_newer_than(){
+    local dir="$1"
+    local than="$2"
+    [[ ! -d "$dir" ]] && return 1
+    for f in "$dir/"*.nix ; do
+        [[ "$f" -nt "$than" ]] && return 0
+    done
+    return 1
+}
+
+
 use_flake() {
   flake_expr="${1:-.}"
   flake_dir="${flake_expr%#*}"
   watch_file "$flake_dir/"flake.nix
   watch_file "$flake_dir/"flake.lock
+  nix_util_dir="$flake_dir/"nix
```

Comment:
I wonder if we should just make this directory configurable? Assuming `nix` sounds like a good default to me.